pkg/helmer/helmer.go

```diff
@@ -33,8 +35,15 @@ import (
 func DefaultSettings() *cli.EnvSettings {
 	s := cli.New()
 
-	s.RepositoryConfig = "/cache/helm/repositories/config.yaml"
-	s.RepositoryCache = "/cache/helm/cache"
+	cacheDir, err := os.UserCacheDir()
+	if err != nil {
+		// Expected path for SRO in container
+		cacheDir = "/root/nonroot/.cache"
```

Comment:
That location is rather specific to having a `nonroot` user on the system.
We create that user in the Dockerfile, so `os.UserCacheDir` should work there. Hence I am not sure that this is a sensible default value.
I would rather use [`os.TempDir`](https://pkg.go.dev/os#TempDir) to build some path, or return the error. We only use that function in initialization code, so I think it might be OK to fail. WDYT?

Reply:
Went with returning an error